src/common/remoteExtension.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {ExtensionMessage, MessagingChannel} from "./extensionMessaging";
+import {InterProcessMessageSender} from "./interProcessMessageSender";
+
+export class RemoteExtension {
```

Comment:
All the methods in this class are 1 line long. Why did we need to refactor that code into a new class?